File: PHP/8php 1/goals.php
<?php
/*
|--------------------------------------------------------------------------
| Grasshopper - Messi goals function
|--------------------------------------------------------------------------
|
*/

function goals (int $laLigaGoals, int $copaDelReyGoals, int $championsLeagueGoals) : int {
   return $laLigaGoals + $copaDelReyGoals + $championsLeagueGoals;
}


function goals(int ...$args): int {
   return array_sum($args);
}


function goals() {
   return array_sum(func_get_args());
}


print_r(goals(0, 0, 0) . ' ' . 0 . '
');
print_r(goals(43, 10, 5) . ' ' . 58 . '
');

File: Python/8py 2/goals.py
"""
|--------------------------------------------------------------------------
| Grasshopper - Messi goals function
|--------------------------------------------------------------------------
|
"""

def goals(laLiga, copaDelRey, championsLeague):
   return laLiga+copaDelRey+championsLeague


goals = lambda x, y, z: sum((x, y, z))


def goals(*a):
   return sum(a)


print(goals(0, 0, 0), 0)
print(goals(5, 10, 2), 17)